src/containers/MainMenu/SettingsMenu.js

```diff
@@ -8,6 +8,15 @@ import { actionCreators as mockActions } from '../../ducks/modules/mock';
 import { actionCreators as dialogsActions } from '../../ducks/modules/dialogs';
 import { actionCreators as deviceSettingsActions } from '../../ducks/modules/deviceSettings';
 
+const personEntry = (protocol) => {
```

Comment:
I think this should live elsewhere - it seems useful generally?

Reply:
Do you think this 'entry' (`[key, val]`) return type would be expected (i.e., good for a utility function)? I don't love it if we're going to reuse the func, but I also don't want to introduce another arbitrary structure.

Reply:
This is now exposed as `getNodeEntryForCurrentPrompt` in the session selector. Return type is Array as before.